internals/cli/cmd_run.go

```diff
@@ -275,3 +283,109 @@ func convertArgs(args [][]string) (map[string][]string, error) {
 
 	return mappedArgs, nil
 }
+
+func maybeCopyPebbleDir(srcDir, destDir string) error {
+	if srcDir == "" {
+		return nil
+	}
+	dirEnts, err := os.ReadDir(destDir)
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
+		return err
+	} else if len(dirEnts) != 0 {
+		// Skip non-empty dir.
+		return nil
+	}
+	fsys := os.DirFS(srcDir)
+	// TODO: replace with os.CopyFS
+	err = _go_os_CopyFS(destDir, fsys)
+	if err != nil {
+		return fmt.Errorf("cannot copy %q to %q: %w", srcDir, destDir, err)
+	}
+	return nil
+}
+
+// Implementation from https://go-review.googlesource.com/c/go/+/558995 until accepted into go.
```

Comment:
Can we please move from here down to its own file so it's super-easy to delete later? Maybe `copyfs.go`.